experimental/packages/opentelemetry-instrumentation/package.json

```diff
@@ -74,7 +74,6 @@
     "@types/shimmer": "^1.2.0",
     "import-in-the-middle": "^1.8.1",
     "require-in-the-middle": "^7.1.1",
-    "semver": "^7.5.2",
```

Comment:
I wonder if we should include `semver` as a development dependency to be used in tests. We could assert that our semver replacement functions output matches the `semver` package.

Reply:
We already have imported tests from `semver` package and test fixtures also include expected result (result of the actual `semver` package). That is the reason that why I have removed `semver` package dependency even from dev dependencies.